src/ResourceManager/Storage/Commands.Management.Storage.Test/App.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime> 
+      <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1"> 
+          <dependentAssembly> 
+              <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" /> 
```

Comment:
I think thsi should be 10.0 if using the latest version of the .net test library